src/components/language-select/template.njk

```diff
@@ -1,5 +1,5 @@
 {% if params.language === 'en' %}
-  <nav class="hmrc-language-select" aria-label="Language switcher">
+  <nav class="hmrc-language-select govuk-!-display-none-print" aria-label="Language switcher">
```

Comment:
worth adding tests for these?

Reply:
@ellamdav Could do, could do... Seems like it's just checking the static template rather than any logic, but can add if that's what we think is best?

Reply:
It feels like double-entry coding in a way, but that's not such a bad thing - eg. someone could refactor the template to dry it up and this might get lost, but visually would still all look OK 🤷